### PR TITLE
Codemirror graphql

### PR DESCRIPTION
**Pull request by rajusah15**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: c/codemirror-graphql/codemirror-graphql_rhel_8.3.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Tested on     : UBI 8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Raju Sah <Raju.Sah@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
```
Comment (kandarpamalipeddi) [inline c/codemirror-graphql/codemirror-graphql_rhel_8.3.sh:14]: Please updated header as per the template.

File: c/codemirror-graphql/codemirror-graphql_rhel_8.3.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#Variables
+REPO=https://github.com/graphql/graphiql.git
+PACKAGE_VERSION=2.0.1
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
+dnf update -y
+dnf module install -y nodejs:12
+#install dependencies
+dnf install -y git 
+#export the path
```
Comment (kandarpamalipeddi) [inline c/codemirror-graphql/codemirror-graphql_rhel_8.3.sh:30]: Remove this comment.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]
Please remove a/airline/airline_rhel_8.3.sh in this pull request.

**Comment by rajusah15** [comment]
New PR has been raised for this package , so i am closing this one.

**Pull request closed without merging**